paper/src/main/resources/plugin.yml

```diff
@@ -7,6 +7,7 @@ authors:
   - "sulu"
   - "ocelotpotpie"
   - "pop4959"
+  - "whiler-sesame"
```

Comment:
nitpick: not to downplay the contribution in any way, but unless you are a *regular* contributor to this plugin please do not add yourself to the authors. I'll make sure @ocelotpotpie credits you properly in the update notes whenever he gets around to releasing this.